src/components/Item/VisualizationItem/Item.js

```diff
@@ -237,9 +237,9 @@ export class Item extends Component {
                     <a
                         href={pluginManager.getLink(this.props.item, this.d2)}
                         style={{ height: 16 }}
-                        title={`View in ${
-                            itemTypeMap[this.props.item.type].appName
-                        } app`}
+                        title={`View in ${itemTypeMap[
```

Comment:
`i18n`? I don't see this string in [en.pot](https://github.com/dhis2/dashboards-app/blob/868a8e90bc2103efcc9480552e0853ea301971f0/i18n/en.pot)

Reply:
Fixed